Automattic/wp-calypso
# Update Backup pending screen design and copy

### Changes proposed in this Pull Request

This PR updates the Backup pending screen to prevent users from contacting support while the site first backup is being made.

Fixes 1197351014149633-as-1197360555544595

### Testing instructions

- Download the PR
- Run Calypso with `yarn start`
- Run Jetpack cloud concurrently with `yarn start-jetpack-cloud-p`
- Create a new self-hosted Jetpack site, and purchase Backup or a plan that includes Backup
- Visit `/backup/:site`
- Check that you see the new screen in both environments
- Check that the copy matches the captures below (watch out the difference between cloud and Calypso), and that the links redirect to the proper places

### Screenshots

#### Jetpack cloud
_Desktop_
<img width="724" alt="Screen Shot 2020-10-06 at 1 14 41 PM" src="https://user-images.githubusercontent.com/1620183/95237568-64c59c00-07d6-11eb-80af-0bf427bc4e43.png">
_Mobile_
<img width="395" alt="Screen Shot 2020-10-06 at 1 15 00 PM" src="https://user-images.githubusercontent.com/1620183/95237570-64c59c00-07d6-11eb-9b62-e3a1aef90db6.png">


#### Calypso
_Desktop_
<img width="757" alt="Screen Shot 2020-10-06 at 1 14 05 PM" src="https://user-images.githubusercontent.com/1620183/95237533-59727080-07d6-11eb-8d2b-318369f9b5b6.png">


_Mobile_
<img width="394" alt="Screen Shot 2020-10-06 at 1 14 15 PM" src="https://user-images.githubusercontent.com/1620183/95237557-6000e800-07d6-11eb-930c-32e2e70d96d8.png">
